src/utils.ts

```diff
@@ -126,6 +124,58 @@ export function equal(target: ObjectInterface, base: ObjectInterface): boolean {
 	return every(target, (v, k) => v === base[k]);
 }
 
-export function toFixed(num: number) {
-	return Math.round(num * FIXED_DIGIT) / FIXED_DIGIT;
+let lastRoundNum = -1;
+let roundNumFunc = null;
+
+export function roundNumber(num: number, roundUnit: number) {
+	// Cache for performance
+	if (!roundNumFunc || lastRoundNum !== roundUnit) {
```

Comment:
@jongmoon 

If there are more than two different axes, there may be a problem with caching them globally.

Reply:
If lastRoundNum is different from **new** roundUnit. it will generate new function. But there may be no performance benefit by caching it. Do you mean this is problem?